src/lua/api-gateway/validation/oauth2/oauthTokenValidator.lua

```diff
@@ -41,24 +41,24 @@
 local BaseValidator = require "api-gateway.validation.validator"
 local cjson = require "cjson"
 
-local _M = BaseValidator:new()
-
-local RESPONSES = {
-    MISSING_TOKEN = { error_code = "403010", message = "Oauth token is missing" },
-    INVALID_TOKEN = { error_code = "401013", message = "Oauth token is not valid" },
-    -- TOKEN_MISSMATCH is reserved for classes overwriting the isTokenValid method
-    TOKEN_MISSMATCH = { error_code = "401014", message = "Token not allowed in the current context" },
-    SCOPE_MISMATCH = { error_code = "401015", message = "Scope mismatch" },
-    UNKNOWN_ERROR = { error_code = "503010", message = "Could not validate the oauth token" }
-}
+local _M = BaseValidator:new({
+    RESPONSES = {
```

Comment:
these responses are usually  converted by [validatorsHandlerErrorDecorator.lua](https://github.com/adobe-apiplatform/api-gateway-request-validation/blob/master/src/lua/api-gateway/validation/validatorsHandlerErrorDecorator.lua). What would be a use-case when this validator would need different error messages ?